ibis/backends/tests/test_vectorized_udf.py

```diff
@@ -341,6 +470,27 @@ def test_reduction_udf_destruct_no_groupby(backend, alltypes):
     backend.assert_frame_equal(result, expected)
 
 
+@pytest.mark.only_on_backends(['pandas'])
```

Comment:
Same here, Should work on pyspark

Reply:
This one in pyspark fails with 
`NotImplementedError: Invalid  return type with grouped aggregate Pandas UDFs: StructType(List(StructField(double_col,DoubleType,true),StructField(mean_weight,DoubleType,true))) is not supported`